app/service/login_handlers/default.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import ldap3
+
+from aiohttp import web
+from aiohttp_jinja2 import render_template
+from ldap3.core.exceptions import LDAPAttributeError, LDAPException
+
+from app.service.interfaces.i_login_handler import LoginHandlerInterface
+
+HANDLER_NAME = 'Default Login Handler'
+
+
+class DefaultLoginHandler(LoginHandlerInterface):
+    def __init__(self, services):
+        super().__init__(HANDLER_NAME)
+        self.log = logging.getLogger('default_login_handler')
+        self.services = services
+        self._ldap_config = self.get_config('ldap')
+
+    async def handle_login(self, request, **kwargs):
+        data = await request.post()
+        username = data.get('username')
+        password = data.get('password')
+        if username and password:
+            if self._ldap_config:
+                verified = await self._ldap_login(username, password)
```

Comment:
since we are splitting out login handlers, should ldap be split into its own, or is there a reason to merge the functionality in with the basic auth one?

Reply:
I wanted to leave the ldap stuff alone for now to avoid having any ldap users change their configurations in this PR. I can put up a future PR that will split ldap into a separate handler